### PR TITLE
Support different endpoint for S3 storage

### PR DESCRIPTION
**Pull request by simahawk**
As documented here https://github.com/sorentwo/carrierwave-aws/issues/40 this is the way to support another S3 storage provider.

/cc @sebastienbeau

### CODE REVIEW

File: lib/generators/locomotive/install/templates/carrierwave_aws.rb [hunk 1]
```diff
@@ -22,6 +22,13 @@
       region:             ENV['S3_BUCKET_REGION']
     }
 
+    # Use a different endpoint (eg: another provider such as Exoscale)
+    if ENV['S3_ENDPOINT'].present?
+      config.aws_credentials.config: AWS.config({
```
Comment (simahawk) [inline lib/generators/locomotive/install/templates/carrierwave_aws.rb]: I'm brand new to Ruby: shall I use dict/hash like syntax instead of dotted path?

**Comment by simahawk** [comment]
@did can you tell me what's wrong here? As far as I see the error does not come from my change but I'm not familiar with these builds. 
By the way, master is :red_circle: as well :wink: 

CC @sebastienbeau

**Comment by did** [comment]
hey @simahawk, sorry for the delay. Just for my personal knowledge, Exoscale provides the same API as AWS S3?

**Comment by did** [comment]
based on https://github.com/sorentwo/carrierwave-aws#custom-options-for-s3-endpoint, looks good to me.

**Comment by did** [comment]
Thanks @simahawk

**Comment by simahawk** [comment]
Ciao @did :)

> Just for my personal knowledge, Exoscale provides the same API as AWS S3?

Yes it does, at least for main features.

**Pull request merged**